src/includes/enriching-events/add-attachment/java.mdx

```diff
@@ -0,0 +1,119 @@
+To add an attachment you can either add it to the scope, pass it to any of the `capture` methods, or manipulate the list of attachments in an `EventProcessor` or `beforeSend`.
+
+
+### Passing attachments to capture
+
+You may pass attachments to any of the `capture` methods, e.g. when capturing an Exception:
+
+```java
+import io.sentry.Attachment;
+import io.sentry.Hint;
+import io.sentry.Sentry;
+
+Sentry.captureException(new IllegalStateException(), Hint.withAttachment("/path/to/file.txt"))
+```
+
+```kotlin
+import io.sentry.Attachment;
+import io.sentry.Hint;
+import io.sentry.Sentry;
+
+Sentry.captureException(IllegalStateException(), Hint.withAttachment("/path/to/file.txt"))
+```
+
+
+### Adding attachments in EventProcessor
+
+You may also manipulate attachments in `EventProcessor`:
+
+```java
+import io.sentry.Attachment;
+import io.sentry.EventProcessor;
+import io.sentry.Hint;
+import io.sentry.Sentry;
+import io.sentry.SentryEvent;
+import io.sentry.protocol.Message;
+
+public class MyClass {
+  public void myMethod() {
+    // Add an `EventProcessor` to the current scope. Note that
+    // this helper will process *all* future events in this scope.
+    Sentry.configureScope(scope -> {
```

Comment:
Maybe don't use `Sentry.configureScope` here but rather focus on the `EventProcessor` class declaration and usage.
You can mention that event processors can be added in the `Scope#addEventProcessor` and `SentryOptions#addEventProcessor`

Reply:
Also, on Android, there's only one scope anyway.

Reply:
updated